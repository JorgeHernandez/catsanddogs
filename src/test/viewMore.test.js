import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { mount, shallow} from 'enzyme';
import expect from 'expect';
import sinon from 'sinon';
import { Switch, Route, BrowserRouter } from 'react-router-dom';
import { Link } from 'react-router-dom';
import ViewMoreButton from '../components/viewMore_btn';

it('::::autocheck::::', () => {
  expect(1-1).toBe(0);
});

it('view more should call search', () => {
	const spy = expect.createSpy();
	const wrapper = shallow(<ViewMoreButton onViewMore={spy} />); 
    const btn = wrapper.find('#viewMore-btn');
    btn.simulate('click');
	expect(spy).toHaveBeenCalled();
});