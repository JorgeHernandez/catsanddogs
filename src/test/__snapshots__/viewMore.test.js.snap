// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<a
  className="btn"
  href="#"
  id="viewMore-btn"
  onClick={undefined}
>
  View more
</a>
`;
